keyboards/vortex/m0110/ansi/m0110_625_ansi/info.json

```
@@ -0,0 +1,107 @@
+{
+    "manufacturer": "Vortex",
+    "keyboard_name": "M0110-625_ANSI",
+    "maintainer": "Gsky",
+    "bootloader": "wb32-dfu",
+    "bootmagic": {
+        "matrix": [1, 6]
+    },
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "enabled": false
+    },
+    "features": {
+        "audio": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "indicators": {
+        "caps_lock": "C15"
+    },
+    "matrix_pins": {
+        "cols": ["A0", "A1", "A2", "A3", "A4", "A8", "A9", "A10"],
+        "rows": ["B0", "B1", "B2", "B3", "B4", "B5", "B6", "B7", "B8", "B9", "B10"]
+    },
+    "processor": "WB32FQ95",
+    "qmk": {
+        "tap_keycode_delay": 10
+    },
+    "url": "",
+    "usb": {
+        "device_version": "0.0.3",
+        "pid": "0xE39E",
+        "vid": "0x342D"
+    },
```

migrated from `rules.mk`
```suggestion
    },
    "eeprom": {
        "driver": "wear_leveling"
    },
```